### PR TITLE
Enable http while running commands is not available

### PR DESCRIPTION
**Pull request by tgodzik**
Confirmed that it works with Eclipse.

### CODE REVIEW

File: lsp-scala-plugin/src/main/scala/com/idiomaticsoft/lsp/scala/ScalaLSPPlugin.scala [hunk 1]
```diff
@@ -54,7 +54,7 @@ class ScalaLSPPlugin extends AbstractUIPlugin {
 		wc.setAttribute(IJavaLaunchConfigurationConstants.ATTR_MAIN_TYPE_NAME, "scala.meta.metals.Main")
 		wc.setAttribute(IJavaLaunchConfigurationConstants.ATTR_DEFAULT_CLASSPATH, false)
 		wc.setAttribute(DebugPlugin.ATTR_CAPTURE_OUTPUT, false)
-		val vmParams = "-XX:+UseG1GC -XX:+UseStringDeduplication -Xss4m -Xms100m -Xmx2G"
+		val vmParams = "-XX:+UseG1GC -XX:+UseStringDeduplication -Xss4m -Xms100m -Xmx2G -Dmetals.http=true"
 		wc.setAttribute(IJavaLaunchConfigurationConstants.ATTR_VM_ARGUMENTS, wc.getAttribute(IJavaLaunchConfigurationConstants.ATTR_VM_ARGUMENTS, vmParams))
```
Comment (mundacho) [inline lsp-scala-plugin/src/main/scala/com/idiomaticsoft/lsp/scala/ScalaLSPPlugin.scala]: I think you need to add a new line:

wc.setAttribute(IJavaLaunchConfigurationConstants.ATTR_PROGRAM_ARGUMENTS,  "-Dmetals.http=true")

The attribute is not for the VM but for the program

Comment (mundacho) [reply lsp-scala-plugin/src/main/scala/com/idiomaticsoft/lsp/scala/ScalaLSPPlugin.scala]: Also, you can modify all these parameters to test in the Run Configuration dialogs for metals

Comment (tgodzik) [reply lsp-scala-plugin/src/main/scala/com/idiomaticsoft/lsp/scala/ScalaLSPPlugin.scala]: Tried that and it didn't seem to work when -D is put after the name of the Main class.

Comment (mundacho) [reply lsp-scala-plugin/src/main/scala/com/idiomaticsoft/lsp/scala/ScalaLSPPlugin.scala]: ok, so these parameters are really properties that are set in the VM

**Pull request merged**